frontend/src/components/Gym/GymKeyWith.tsx

```diff
@@ -52,6 +52,21 @@ interface Props {
   avatar?: string
 }
 
+/**
+ *
+ * @param name (string)
+ * @param handle (string) user's telegram handle
+ * @param avatar (string, optional) user's profile picture. The string is of the form url
+ * @returns user's profile picture, name and link to his/her telegram
+ *
+ * @example
+ * user called Shaun and has no profile picture
+ * component will display the default profile picture, his name Shaun as well as his
+ * telegram handle. Clicking on the telegram handle redirects the user to key holder's telegram
+ * <GymKeyWith name={"thomaslim"} handle={"@thomas123"} avatar={""} />
```

Comment:
btw JSX syntax string params don't need curly braces, just double quotes can alr
```suggestion
 * <GymKeyWith name="thomaslim" handle="@thomas123" avatar="" />
```